packages/types/src/procaptcha/manager.ts

```diff
@@ -41,3 +41,23 @@ export type ProcaptchaOutput = zInfer<typeof ProcaptchaOutputSchema>
 export const ProcaptchaResponse = object({
     [ApiParams.procaptchaResponse]: ProcaptchaOutputSchema,
 })
+
+/**
+ * A set of callbacks called by Procaptcha on certain events. These are optional
+ * as the client can decide which events they wish to listen for.
+ */
+export type ProcaptchaCallbacks = Partial<ProcaptchaEvents>
+
+/**
+ * A list of all events which can occur during the Procaptcha process.
+ */
+export interface ProcaptchaEvents {
```

Comment:
where were these before being moved to here?

Reply:
Defined in Procaptcha package. Made it a nightmare for code splitting and reusing ProcaptchaEvents outside of image captcha (which wasn't really much of an issue prior to pow/frictionless)

Basically just keeping types in the types pkg and avoiding circular deps :)

Reply:
okay sounds good